server/routes/visualise.js

```diff
@@ -3,7 +3,36 @@ const whatDoesItAllMean = require('../lib/report');
 
 module.exports = function (req, res){
 
-	whatDoesItAllMean()
+	const visRequests = [whatDoesItAllMean.insights.allValues(), whatDoesItAllMean.insights.historicalDomain('www.theguardian.com')];
+
+	Promise.all(visRequests)
+		.then(results => {
+
+			// debug(`First row of response\n${results[0][0]}\n${results[1][0]}`);
+
+			debug(results[1]);
+
+			res.render('main', {
+				partial : 'visualise',
+				results : {
+					dayResults : results[0],
+					historical : results[1]
+				}
+			});
+
+		})
+		.catch(err => {
+
+			res.render('main', {
+				partial : 'visualise'
```

Comment:
It is empty, we need a 404 page or something similar really. Maybe a 'Sorry , we have no data right now' partial?